bugsnag-android-core/src/main/java/com/bugsnag/android/BugsnagEventMapper.kt

```diff
@@ -11,6 +11,19 @@ internal class BugsnagEventMapper(
     private val logger: Logger
 ) {
 
+    @Suppress("UNCHECKED_CAST")
+    internal fun convertBreadcrumbInternal(breadcrumb: Map<String, Any?>): BreadcrumbInternal {
+        return BreadcrumbInternal(
+            breadcrumb.readEntry("name"),
+            breadcrumb.readEntry<String>("type").let { type ->
+                BreadcrumbType.fromDescriptor(type)
+                    ?: throw IllegalArgumentException("unknown BreadcrumbType: '$type'")
```

Comment:
I'd coerce unknown types to `manual`, that's how Potential Shenanigans are handled on other platforms / in request data.